### ISSUE TITLE
It is not possible to replace video in tags a-sky, a-videosphere, a-video

### ISSUE THREAD

**Issue by dhaiduk** [8thwall/web#146]
It is not possible to replace the video in tags a-sky, a-videosphere, a-video
But if just use a-frame, then the video can be replaced with another video.

**Comment by atomarch** [comment]
Yes, you can do this.  Here is one example: https://www.8thwall.com/playground/portal-stream
